astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -85,80 +95,134 @@ class galactocentric_frame_defaults(ScienceState):
     """
 
     _latest_value = 'v4.0'
-    _references = None
     _value = None
+    _references = None
+    _state = dict()  # all other data
+
+    @classproperty  # read-only
+    def value(cls):
```

Comment:
Internally, this info has to be saved as `._value` to work with `ScienceState`, but I wonder if we want to expose this as `parameters_name` or something more descriptive? No strong feelings though..

Reply:
I think I made the same comment but @nstarman had a good answer - which I've forgotten... Was it something about possibly having `uncertainties` in the future?  I still wonder the same. If we change, I'd do `parameters` (or perhaps `attributes` - since in the end these become attributes to the class? I think I like parameters better, given analogy with a model).

Reply:
I agree that `parameters` is a better name than `value`. With the next PR, any key in `_state` will automatically be accessible as attributes. I suppose it's not too bad to have parameters -> `_value`. With both of you in agreement, I'll change it.